addons/web/static/src/js/public/public_root.js

```diff
@@ -84,6 +84,11 @@ var PublicRoot = publicWidget.RootWidget.extend({
             $('input, textarea').placeholder();
         }
 
+        // Library clears the wrong date format so just ignore error
+        $('#wrapwrap').on('error.datetimepicker', function (ev) {
+            return false;
+        });
```

Comment:
- Maybe write it `this.$el.children()` instead of `$('#wrapwrap')`, that way we'll target all top level areas once and for all?
- Just for info: We can try to use more ES6 so I would have written `() => false` and make this a one-liner
- but: even if outside of the events key I would have made this a dedicated event handler users can still override if needed, just for the convention and consistence: `this.$el.children().on('error.datetimepicker', this._onDatetimePickerError.bind(this));`

Reply:
Changed.